docs/source/examples/checks/distribution/whole_dataset_drift.ipynb

```diff
@@ -5,15 +5,66 @@
    "id": "104dbd2d",
```

Comment:
<p>First paragraph - link to drift notebook?</p><p> </p><blockquote>drift check, a multivariate drift</blockquote><p>the "a" should be replaced with "the"</p><p> </p><p>The term "a known distribution" is confusing, and then the next paragraph explains it well, so maybe find a way to merge it.</p><p> </p><p>Don't think that we need to go into this amount of details here (like type of model).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>